.gomarkdoc.yml

```diff
@@ -0,0 +1,4 @@
+excludeDirs: "./cmd/..."
+output: "{{.Dir}}/README.md"
+repository:
+  url: https://github.com/cinar/indicator
```

Comment:
`repository.url` is necessary for forked repository!  Otherwise, in my forked repository, the docs becomes like this.

```
 <a name="Ad"></a>
-## type [Ad](<https://github.com/cinar/indicator/blob/master/volume/ad.go#L20-L23>)
+## type [Ad](<https://github.com/sunglim/indicator/blob/master/volume/ad.go#L20-L23>)
```